### PR TITLE
[9.0][NEW] stock_picking_report_custom_description: New module WIP

### PR DESCRIPTION
**Pull request by carlosdauden**
New report to print move description in delivery split.
Forced allways to print from moves instead of operations.

@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The name should be `stock_picking_report_custom_description`

File: stock_picking_report_custom_description/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+import models
```
Comment (pedrobaeza) [inline stock_picking_report_custom_description/__init__.py]: `from . import...`

File: stock_picking_report_custom_description/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Picking Report Custom Description",
+    "summary": "New report to print move description in delivery split",
+    "version": "9.0.1.0.0",
+    "category": "Stock",
```
Comment (pedrobaeza) [inline stock_picking_report_custom_description/__openerp__.py]: Isn't `Warehouse` the `stock` module category?

File: stock_picking_report_custom_description/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Picking Report Custom Description",
+    "summary": "New report to print move description in delivery split",
```
Comment (pedrobaeza) [inline stock_picking_report_custom_description/__openerp__.py]: `Show moves description in picking reports`

File: stock_picking_report_custom_description/i18n/es.po [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+#: model:ir.actions.report.xml,name:stock_picking_report_custom_description.action_report_picking_description
+msgid "Picking Operations (Description)"
+msgstr "Operaciones de picking (descripción)"
+
+#. module: stock_picking_report_custom_description
+#: model:ir.model.fields,field_description:stock_picking_report_custom_description.field_stock_pack_operation_several_product_description
+msgid "Several product description"
+msgstr "Varias descripciones de producto"
+
+#~ msgid "Description"
```
Comment (pedrobaeza) [inline stock_picking_report_custom_description/i18n/es.po]: Why is this here?

File: stock_picking_report_custom_description/i18n/es.po [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+#. module: stock_picking_report_custom_description
+#: model:ir.model,name:stock_picking_report_custom_description.model_stock_pack_operation
+msgid "Packing Operation"
+msgstr "Operación de empaquetado"
+
+#. module: stock_picking_report_custom_description
+#: model:ir.actions.report.xml,name:stock_picking_report_custom_description.action_report_picking_description
+msgid "Picking Operations (Description)"
+msgstr "Operaciones de picking (descripción)"
```
Comment (pedrobaeza) [inline stock_picking_report_custom_description/i18n/es.po:43]: s/picking/albarán

File: stock_picking_report_custom_description/views/report_deliveryslip.xml [hunk 6]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <template id="report_delivery_document_description" inherit_id="stock.report_delivery_document" primary="True">
```
Comment (pedrobaeza) [inline stock_picking_report_custom_description/views/report_deliveryslip.xml]: Isn't better to overwrite standard report with this one?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: stock_picking_report_custom_description/i18n/es.po (hunk 5 above)
Comment (carlosdauden) [reply stock_picking_report_custom_description/i18n/es.po:43]: This Is the same translation of original report name

File: stock_picking_report_custom_description/views/report_deliveryslip.xml (hunk 6 above)
Comment (carlosdauden) [reply stock_picking_report_custom_description/views/report_deliveryslip.xml]: In this way we have more flexibility at no cost

**Comment by carlosdauden** [comment]
I think it does serve:
https://github.com/OCA/stock-logistics-reporting/pull/32/files#diff-2b79a6178ed01a822c4c4f1fa17bdc9eR25

File: stock_picking_report_custom_description/README.rst [hunk 7]
```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Stock Picking Report Custom Description
+=======================================
+
+This module allows to print moves description in picking reports.
+
```
Comment (MiquelRForgeFlow) [inline stock_picking_report_custom_description/README.rst:10]: Please, could you add <code>Usage</code> section?

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: stock_picking_report_custom_description/i18n/es.po (hunk 5 above)
Comment (pedrobaeza) [reply stock_picking_report_custom_description/i18n/es.po:43]: Original translation is incorrect then :wink:

**Pull request closed without merging**